copter/source/docs/zigzag-mode.rst

```diff
@@ -30,3 +30,21 @@ When flown manually ZigZag uses :ref:`Loiter mode's <loiter-mode>`  parameters.
 
 When autonomously flying from one side of the field to the other, the :ref:`Auto mode <auto-mode>` parameters are used.  For example the top speed can be configured with :ref:`WPNAV_SPEED <WPNAV_SPEED>`..
 
+Automatic Operation
+===================
+
+.. note:: this function must be enabled with :ref:`ZIGZ_AUTO_ENABLE<ZIGZ_AUTO_ENABLE>` = 1, and rebooted, before any parameters are visible and the feature enabled.
```

Comment:
I don't think it needs a reboot for it to take effect.  A parameter refresh will be required to show the additional parameters.

Reply:
changed